packages/terra-card/src/CardPadding.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Card.scss';
+
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes
+   */
+  children: PropTypes.node,
+  /**
+   * Provides 10px of themeable padding vertical
+   */
+  hasPaddingVertical: PropTypes.bool,
+  /**
+   * Provides 10px of themeable padding horizontal
+   */
+  hasPaddingHorizontal: PropTypes.bool,
+};
+
+const defaultProps = {
+  hasPaddingHorizontal: false,
+  hasPaddingVertical: false,
+};
+
+
+const CardPadding = ({
```

Comment:
I actually I'll partially take that back. If we can assume cascaded styling then we can use global utilities like that.  But I don't think we can make that assumption quite yet....

```css
:root {
  --global-padding: 10px;
}
.global-padding {
   padding: var(--global-padding);
}

.card {
 --card-padding: --global-padding;
  padding: var(--card-padding);
}
```